src/components/app/details/appDetails/SourceInfo.tsx

```diff
@@ -278,66 +274,12 @@ export function SourceInfo({
                                     )}
                                 </div>
                             </div>
-                            <div
```

Comment:
is removal of complete code expected?

Reply:
Yes extracted out in a separate component named `DeploymentStatusCard`